data/json/items/comestibles/egg.json

```diff
@@ -490,315 +490,270 @@
   {
     "type": "COMESTIBLE",
     "id": "egg_brown_trout",
-    "name": { "str_sp": "roe" },
```

Comment:
Are you sure name isn't a required field?  Cause that error report sure seems to be suggesting it is.

Reply:
It is. Waiting for #71199